chromadb/api/client.py

```diff
@@ -402,13 +403,21 @@ def set_database(self, database: str) -> None:
     def _validate_tenant_database(self, tenant: str, database: str) -> None:
         try:
             self._admin_client.get_tenant(name=tenant)
+        except requests.exceptions.ConnectionError:
+            raise ValueError(
+                "Could not connect to a Chroma server. Are you sure it is running?"
```

Comment:
@HammadB, wdyt about adding the URL we're trying to connect to? I'd give users a better experience in case they have a typo in the URL.